CONTRIBUTE.rst

```diff
@@ -71,19 +71,19 @@ If you want to add a new connector source, you need to:
 2. implement a class that implements all methods described in `connectors.source.BaseDataSource`
 3. add a unit test in `connectors/sources/tests <connectors/sources/tests>`_ with +90% coverage
 4. declare your connector in `config.yml <config.yml>`_ in the `sources` section
-5. declare your dependencies in `requirements.txt <requirements.txt>`_
+5. declare your dependencies in `requirements.txt <requirements.txt>`_. Make sure you pin these dependencies
 6. make sure you use an async lib for your source. If not possible, make sure you don't block the loop
 7. when possible, provide a docker image that runs the backend service, so we can test the connector
 8. if you can't provide a docker image, provide the credentials needed to run against a service
-9. The backend needs to return 10,001 documents.
+9. the test backend needs to return 10,001 documents due to internal implementation details of the connector - it batches the data into chunks 0f 10,000 items
```

Comment:
this is not the reason actually. I added 10,001 because 10k is the default size limit for pagination, so having the whole flow work for +1 seems like a decent default for folks building and trying out their connectors https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-max-result-window

Reply:
Oh right! I've changed the comment to:
> the test backend needs to return 10,001 documents due to 10,000 being a default size limit for Elasticsearch pagination. Having 10,001 documents returned from the test backend will help testing connector more deeply

Does it look better?

Reply:
thx!